Integration Examples/eAPI v1.6/php/crypto.php

```diff
@@ -91,14 +91,12 @@ function signPaymentInitData( $data, $privateKey, $privateKeyPassword) {
 	$data2Sign = $data["merchantId"] . "|" .  $data["orderNo"] . "|" . $data["dttm"] . "|" . $data["payOperation"] . "|" . $data["payMethod"] . "|" . $data["totalAmount"]
 		."|". $data["currency"] ."|". $data["closePayment"]  . "|". $data["returnUrl"] ."|". $data["returnMethod"] . "|" . $cart2Sign . "|" . $data["description"];
 
-	$merchantData = $data["merchantData"];
-	if(!is_null($merchantData)) {
-		$data2Sign = $data2Sign . "|" . $merchantData;
+	if(!isset($data["merchantData"])) {
```

Comment:
if(isset($data["merchantData"])) {

Reply:
@dmarek ^^ Gentle reminder, fixed in 3a7b117 +via #619